### PR TITLE
feat: add filters to `ChaindeskRetriever`

### PR DESCRIPTION
**Pull request by antoinewg**
reference: https://docs.chaindesk.ai/api-reference/endpoint/datastores/query

cc @gmpetrov

### CODE REVIEW

File: langchain/src/retrievers/chaindesk.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ export interface ChaindeskRetrieverArgs
     BaseRetrieverInput {
   datastoreId: string;
   topK?: number;
+  filters?: Record<string, unknown>;
```
Comment (jacoblee93) [inline langchain/src/retrievers/chaindesk.ts:10]: We generally call them just `filter` (no s)

**Comment by jacoblee93** [comment]
Seems good and backwards compatible to me, see comment!

**Comment by jacoblee93** [comment]
Can you run `yarn format`?

**Comment by antoinewg** [comment]
should be good now @jacoblee93 :)

**Comment by jacoblee93** [comment]
Thank you!

**Pull request merged**